apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -279,7 +283,7 @@ SOMAExperimentAxisQuery <- R6::R6Class(
         x = self$var_df$attrnames(),
         y = features_index
       )
-      if (!(isFALSE(var_column_names) || rlang::is_na(var_column_names))) {
+      if (!(isFALSE(var_column_names) || is_na(var_column_names))) {
```

Comment:
(FWIW I'd keep the prefix to make it clear where it comes from.  You need the NAMESPACE nag to avoid R CMD check from coming after you...)